### ISSUE TITLE
Dino doesn't seem to find my existing contacts

### ISSUE THREAD

**Issue by bochecha** [dino/dino#204]
I just started Dino for the first time and was greeted by the empty window, prompting me to add an account. I did so, then tried to start a conversation.

The modal dialog which appeared at this point was completely empty. Clicking on the [+] button I could add one of my existing contacts and start a chat.

However, I would have expected the list to be prepopulated by my existing contacts, so I could just have selected one of them to start chatting.

This is with a build of f6ac5bbd26638412a2289fd1d28ef12de1d7e8b5.

**Comment by fiaxh** [comment]
Does the accounts window state `Connected` below your account JID? Which client did you use before?

When you start dino with `--print-xmpp="iq"`, some of the traffic will be printed to console. Please search for the first occurrence of `xmlns='jabber:iq:roster'`. This should be something like 

```xml
<iq id='cd9ca32c-4f4f-45f1-8d8d-6a3a5e7a123f' type='get'>
  <query xmlns='jabber:iq:roster' />
</iq>
```

When you then search for the id (`cd9ca32c-4f4f-45f1-8d8d-6a3a5e7a123f`), you should find one other node. What does it look like?

**Comment by bochecha** [comment]
> Does the accounts window state Connected below your account JID?

It does.

> Which client did you use before?

I have used, and still use, Empathy. (but I'd really like to move away from it >_<)

---

This is what I found for the output you requested:

```
XMPP OUT [bochecha@daitauha.fr]
<iq id='5b988743-6fed-4b3a-97fa-5b5bbd7fdc03' type='get'>
  <query xmlns='jabber:iq:roster' ver='10' />
</iq>

[... snip ...]

XMPP IN [bochecha@daitauha.fr]
<iq id='5b988743-6fed-4b3a-97fa-5b5bbd7fdc03' type='result' to='bochecha@daitauha.fr/dino.3c15eb4d' />
```

Let me know if there's anything else I can try. 🙂

**Comment by bochecha** [comment]
Oh wow, I feel pretty stupid.

I have two email accounts on the same host, something like: user1@example.com and user2@example.com

Both also are Jabber IDs, but I only use user1@example.com for Jabber. (I use them both for email though)

Somehow, when I tried Dino out, I used user2@example.com.

That is why Dino couldn't see any of my contacts: I don't have any on this account! 🙄 

You asked me in a private chat to try deleting my account and readding it, and that's when I realized my mistake.

Now, using the right Jabber ID, I have all my contacts as expected.

I hope I didn't waste too much of your time on this, but if I did I'm sorry.

/me goes to hide quietly in a corner
